app/appointments/utils/createCalendarEvent.ts

```diff
@@ -0,0 +1,43 @@
+import * as ics from "ics"
+import { Appointment } from "../types"
+
+function durationToIcsDurationObject(duration: number): ics.DurationObject {
+  const durationInMinutes = duration / (60 * 10000)
+  return {
+    hours: Math.floor(durationInMinutes / 60),
+    minutes: Math.floor(durationInMinutes % 60),
+  }
+}
+
+export function createCalendarEvent(appointment: Appointment): string {
+  const { error, value } = ics.createEvent({
+    start: [
+      appointment.start.getFullYear(),
+      appointment.start.getMonth() + 1,
+      appointment.start.getDay(),
```

Comment:
getDay only returns the day of a week ([link](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay)) which might be 0.
```suggestion
      appointment.start.getDate(),
```

Reply:
Good catch!